hoomd/md/force.py

```diff
@@ -210,6 +211,12 @@ def set_forces(self, timestep):
     property name with either the prefix ``ghost_`` of the suffix
     ``_with_ghost``.
 
+    Note:
+        If updating the torque array in the set_forces method, make sure to set
+        the `aniso` property of the custom force object to `True` so the
+        simulation knows to update the rotational degrees of freedom in the
+        system.
```

Comment:
This makes it sound like users should set `aniso` on a step by step basis. However, the implementation assumes that `isAnisotropic` is a static property determined at class instantiation. 

`aniso` should not be a modifiable parameter dict entry, it should be set in `__init__` and remain static.

I would then reword this section of the docs to read:
"Set `aniso=True` if your custom force produces non-zero torques on particles."